src/region/RegionCreator.scss

```diff
@@ -6,7 +6,3 @@ $region_cursor_32_3x: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGAAAABgCAY
     cursor: url($region_cursor_32) 16 16, crosshair; /* Legacy */
     cursor: image-set(url($region_cursor_32) 1x, url($region_cursor_32_2x) 2x, url($region_cursor_32_3x) 3x) 16 16, crosshair; /* Webkit */ /* stylelint-disable-line */
 }
-
-.ba-RegionCreator-rect {
-    will-change: height, transform, width;
```

Comment:
Removing this and the translate3d style actually seemed to reduce reflows and cpu usage, at least in Chrome, probably because we're not longer using `transform` at all.